Gemfile

```diff
@@ -2,10 +2,8 @@ source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
 group :development, :test do
   # Pinning `rake` because `v11` doesn't support Ruby 1.8.7
-  gem 'rake', '10.5.0'
+  gem 'rake'
   gem 'puppetlabs_spec_helper', :require => false
-  # Pinning due to bug in newer rspec with Ruby 1.8.7
-  gem 'rspec-core', '3.1.7'
```

Comment:
I run into some issues while running the tests locally. For debugging I removed the pinning.

Reply:
We shouldn't need to support such an old Ruby version anymore, so this should be ok.

Reply:
I rebased against the release branch to get all this fixed. I hope that was okay.